src/main/java/com/faforever/client/game/TeamCardController.java

```diff
@@ -59,25 +59,31 @@ public static void createAndAdd(ObservableMap<? extends String, ? extends List<S
 
       TeamCardController teamCardController = uiService.loadFxml("theme/team_card.fxml");
       teamCardController.setPlayersInTeam(entry.getKey(), players,
-          player -> new Rating(player.getGlobalRatingMean(), player.getGlobalRatingDeviation()));
+          player -> new Rating(player.getGlobalRatingMean(), player.getGlobalRatingDeviation()), true);
       teamsPane.getChildren().add(teamCardController.getRoot());
     }
   }
 
-  public void setPlayersInTeam(String team, List<Player> playerList, Function<Player, Rating> ratingProvider) {
+  public void setPlayersInTeam(String team, List<Player> playerList, Function<Player, Rating> ratingProvider, boolean useRoundedRating) {
```

Comment:
`useRoundedRating` can then be replaced by the new enum. Note also how the enum allows multiple values while boolean only allow two, and how passing `RatingMode.ROUNDED` is more expressive than passing `true`.